packages/vaul-vue/package.json

```diff
@@ -15,9 +15,10 @@
     "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore",
     "format": "prettier --write src/"
   },
-  "dependencies": {
-    "radix-vue": "^1.2.5",
-    "vue": "^3.4.5"
+  "peerDependencies": {
+    "radix-vue": "^1.2.0",
```

Comment:
My thought was to bundle the radix-vue dependency so that it works standalone. Thoughts?

Reply:
Ow yes you right 👍